web/src/utils/helpers.ts

```diff
@@ -53,6 +53,19 @@ export function comparePipelines(a: Pipeline, b: Pipeline): number {
   return (b.created_at || -1) - (a.created_at || -1);
 }
 
+/**
+ * Compare two pipelines by the status.
+ * Giving pending, running, or started higher priority than other status
+ * @param {Object} a - A pipeline.
+ * @param {Object} b - A pipeline.
+ * @returns {number}
+ */
+export function comparePipelinesWithStatus(a: Pipeline, b: Pipeline): number {
```

Comment:
do we have a testframework ?

That should have a simple coverage :)

Reply:
Can we merge it or do we want to add one first?

Reply:
-> #2694 

Input is welcome :)